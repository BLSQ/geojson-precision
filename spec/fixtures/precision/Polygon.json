{
    "type": "Polygon",
    "coordinates": [
      [
        [
          -10.546875,
          35.17380831799959
        ],
        [
          -10.546875,
          44.59046718130883
        ],
        [
          2.4609375,
          44.59046718130883
        ],
        [
          2.4609375,
          35.17380831799959
        ],
        [
          -10.546875,
          35.17380831799959
        ]
      ]
    ]
  }