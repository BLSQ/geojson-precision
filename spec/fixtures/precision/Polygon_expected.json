{
  "type": "Polygon",
  "coordinates": [
    [
      [
        -10.547,
        35.174
      ],
      [
        -10.547,
        44.59
      ],
      [
        2.461,
        44.59
      ],
      [
        2.461,
        35.174
      ],
      [
        -10.547,
        35.174
      ]
    ]
  ]
}